great_expectations_cloud/agent/actions/run_checkpoint.py

```diff
@@ -64,7 +64,7 @@ def run_checkpoint(
     created_resources = []
     for key in validation_results.keys():
         created_resource = CreatedResource(
-            resource_id=validation_results[key].id,
+            resource_id=validation_results[key].id or "",  # id can be None
```

Comment:
Same comment as before. Is this ever None in practice?

Reply:
Probably not. I'm not sure.